examples/js/effects/VREffect.js

```diff
@@ -55,29 +69,59 @@ THREE.VREffect = function ( renderer, onError ) {
 
 	// fullscreen
 
-	var isFullscreen = false;
+	var isPresenting = false;
 
 	var canvas = renderer.domElement;
 	var fullscreenchange = canvas.mozRequestFullScreen ? 'mozfullscreenchange' : 'webkitfullscreenchange';
 
 	document.addEventListener( fullscreenchange, function ( event ) {
 
-		isFullscreen = document.mozFullScreenElement || document.webkitFullscreenElement;
+		if ( deprecatedAPI ) {
+
+			isPresenting = document.mozFullScreenElement || document.webkitFullscreenElement;
+
+		}
+
+	}, false );
+
+	window.addEventListener( 'vrdisplaypresentchange', function ( event ) {
+
+		isPresenting = vrHMD.isPresenting;
 
 	}, false );
 
 	this.setFullScreen = function ( boolean ) {
```

Comment:
understood, that makes sense. so it can be removed more easily in the future, perhaps create a new method, alias the old one and a deprecation warning. similar thing was done to [`zeroSensor`](https://github.com/toji/three.js/blob/webvr_v1/examples/js/controls/VRControls.js#L121-L126).